doc/source/tune/_tutorials/tune-pytorch-lightning.rst

```diff
@@ -102,14 +102,22 @@ The callback just reports some metrics back to Tune after each validation epoch:
    :start-after: __tune_callback_begin__
    :end-before: __tune_callback_end__
 
+Note that we have to explicitly convert the metrics from a tensor to a Python value.
+
 Adding the Tune training function
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Then we specify our training function. Note that we added the ``data_dir`` as a config
-parameter here, even though it should not be tuned. We just need to specify it to avoid
+Then we specify our training function. Note that we added the ``data_dir`` as a
+parameter here to avoid
 that each training run downloads the full MNIST dataset. Instead, we want to access
 a shared data location.
 
+We are also able to specify the number of epochs to train each model, and the number
+of GPUs we want to use for training. We also create a TensorBoard logger that writes
+logfiles directly into Tune's root trial directory - if we didn't do that PyTorch
+Lightning would create subdirectories, and each trial would thus be shown twice in
```

Comment:
should we disable the PTL logger? Otherwise, Tune logs at the same time PTL does - won't this cause duplicate logs?

Reply:
When I tested that yesterday they didn't seem to interfere, but I will make sure this is really the case.

Reply:
ok sounds good; it's probably just safe to disable one

Reply:
So there is no interference between the two loggers - i.e. it is safe to use both of them at the same time, as they log to different namespaces. 
PyTorch Lightning supports custom logs, so should be included. E.g. they log the train and accuracy, too. The only duplicate logs are the evaluation metrics logged by Tune, and even there they use different time scales (tune uses number of epochs, while PTL uses the timestep).
I just pushed a commit to add scoping to the PyTorch Lightning logs so they get grouped in tensorboard. Other than that I think we're ready to go.